Userland/Libraries/LibCore/System.cpp

```diff
@@ -10,6 +10,7 @@
 #include <AK/Vector.h>
```

Comment:
can we mention where in LibCore the hardcoded constant is used? a la "Remove usage of a hardcoded constant in gethostname"